core/vm/contracts.go

```diff
@@ -99,16 +107,21 @@ var PrecompiledContractsBerlin = map[libcommon.Address]PrecompiledContract{
 }
 
 var PrecompiledContractsCancun = map[libcommon.Address]PrecompiledContract{
-	libcommon.BytesToAddress([]byte{0x01}): &ecrecover{},
-	libcommon.BytesToAddress([]byte{0x02}): &sha256hash{},
-	libcommon.BytesToAddress([]byte{0x03}): &ripemd160hash{},
-	libcommon.BytesToAddress([]byte{0x04}): &dataCopy{},
-	libcommon.BytesToAddress([]byte{0x05}): &bigModExp{eip2565: true},
-	libcommon.BytesToAddress([]byte{0x06}): &bn256AddIstanbul{},
-	libcommon.BytesToAddress([]byte{0x07}): &bn256ScalarMulIstanbul{},
-	libcommon.BytesToAddress([]byte{0x08}): &bn256PairingIstanbul{},
-	libcommon.BytesToAddress([]byte{0x09}): &blake2F{},
-	libcommon.BytesToAddress([]byte{0x0a}): &pointEvaluation{},
+	libcommon.BytesToAddress([]byte{1}):                    &ecrecover{},
+	libcommon.BytesToAddress([]byte{2}):                    &sha256hash{},
+	libcommon.BytesToAddress([]byte{3}):                    &ripemd160hash{},
+	libcommon.BytesToAddress([]byte{4}):                    &dataCopy{},
+	libcommon.BytesToAddress([]byte{5}):                    &bigModExp{eip2565: true},
+	libcommon.BytesToAddress([]byte{6}):                    &bn256AddIstanbul{},
+	libcommon.BytesToAddress([]byte{7}):                    &bn256ScalarMulIstanbul{},
+	libcommon.BytesToAddress([]byte{8}):                    &bn256PairingIstanbul{},
+	libcommon.BytesToAddress([]byte{9}):                    &blake2F{},
+	libcommon.BytesToAddress([]byte{20}):                   &pointEvaluation{},
```

Comment:
Please restore the change of the `pointEvaluation` precompile made in PR #7940 (0x0a instead of 20).